### DISCUSSION TITLE
How do I remove a browser from FFupdater?

### DISCUSSION THREAD

**Discussion by Aufschlauer** [Tobi823/ffupdater#463] [Q&A]
I still have Ungoogled Chromium `99.0.4844.51` installed. FFupdater suggest ver. `99.0.4844.51-1` - which is pretty useless to download and install.

So, how can I remove Ungoogled Chromium from the list of updatable browsers?

Thanks.

**Comment by Tobi823** [comment]
You can:
- uninstall the app (e.g. Ungoogled Chromium) or 
- you can disable the background update check for this app (to safe some bandwidth)

Why did I do this? I want to keep FFUpdater as simple as possible. Even after removing and installing should FFUpdater just work out of the box (at least I want/hope so)
